version-stable-r-development.Rmd

```diff
@@ -83,38 +91,42 @@ following setup:
 - No authentication required (local setup).
 - Enable root by setting the environment variable `ROOT` to `TRUE`, so that e.g.
 `sudo apt-get` can be used in RStudio.
-- Use a version-specific port, e.g. `3500` for R 3.5.0, `3610` for R 3.6.1 and
-so on, so that we can use `localhost` for concurrent R version instances.
+- Use a version-specific port, e.g. `4000` for R 4.0.0, `4410` for R 4.4.1 and
+so on, so that we can use `localhost` for concurrent R version instances. 
+We bind the port to localhost (`127.0.0.1:4410`), so it is only accessible locally 
+(see [the Rocker reference](https://rocker-project.org/images/versioned/rstudio.html#disable_auth)).
 - The development code of all relevant projects should live outside the
-container and be shared with it (and possibly many of them), e.g. under
-`~/RStudioProjects` on the host machine and `/home/rstudio/RStudioProjects` in
+container and be shared with it (and possibly multiple other containers), e.g. under
+`~/workspace` on the host machine and `/home/rstudio/workspace` in
```

Comment:
I think it makes more sense to have e.g. `~/workspace/RStudioProjects` mapped to `/home/rstudio/workspace/RStudioProjects`, also in view of `$SHARED_DIR` defined and used in the example and `run_rstudio_ver`.

This way, we are always sure we have something under `workspace` in the container, and "workspace" is a bit generic as a local directory.

Reply:
OK, I had somehow the feeling we always wanted something under `/home/rstudio/workspace`, but there is no real reason for this